src/app/services/post.service.ts

```diff
@@ -11,68 +11,83 @@ interface Options {
   }
 
-  observable(boardID: string, handleAdd: Function, handleModification: Function) {
-    return this.postsCollection.ref.where("boardID", "==", boardID).onSnapshot((snapshot) => {
-      snapshot.docChanges().forEach((change) => {
-        if (change.type === "added") {
-          handleAdd(change.doc.data())
-        } else if (change.type === "modified") {
-          handleModification(change.doc.data())
-        }
-      })
-    })
+  observable(boardID: string,handleAdd: Function,handleModification: Function) {
+    return this.postsCollection.ref
+      .where('boardID', '==', boardID)
+      .onSnapshot((snapshot) => {
+        snapshot.docChanges().forEach((change) => {
+          if (change.type === 'added') {
+            handleAdd(change.doc.data());
+          } else if (change.type === 'modified') {
+            handleModification(change.doc.data());
+          }
+        });
+      });
   }
 
   observeOne(postID: string, handleUpdate: Function, handleDelete: Function) {
-    return this.postsCollection.ref.where("postID", "==", postID).onSnapshot((snapshot) => {
-      snapshot.docChanges().forEach((change) => {
-        let post = change.doc.data()
-        if (change.type == "modified") {
-          handleUpdate(post)
-        } else if (change.type == "removed") {
-          handleDelete(post);
-        }
-      })
-    })
+    return this.postsCollection.ref
+      .where('postID', '==', postID)
+      .onSnapshot((snapshot) => {
+        snapshot.docChanges().forEach((change) => {
+          let post = change.doc.data();
+          if (change.type == 'modified') {
+            handleUpdate(post);
+          } else if (change.type == 'removed') {
+            handleDelete(post);
+          }
+        });
+      });
   }
 
   get(postID: string) {
-    return this.postsCollection.ref.where("postID", "==", postID).get().then((snapshot) => snapshot)
+    return this.postsCollection.ref
+      .where('postID', '==', postID)
+      .get()
+      .then((snapshot) => snapshot);
   }
 
   getAll(boardID: string) {
-    return this.postsCollection.ref.where("boardID", "==", boardID).get().then((snapshot) => snapshot)
+    return this.postsCollection.ref
+      .where('boardID', '==', boardID)
+      .get()
+      .then((snapshot) => snapshot);
   }
 
   getPaginated(boardID: string, opts: Options) {
-    return this.postsCollection.ref.where("boardID", "==", boardID)
-      .orderBy("timestamp")
+    return this.postsCollection.ref
+      .where('boardID', '==', boardID)
+      .orderBy('timestamp')
       .startAfter(opts.lastItem)
       .limit(opts.pageSize)
       .get()
-      .then(data => {
-        let newLastItem = data.docs[data.docs.length - 1]
-        return { newLastItem, data }
-      })
+      .then((data) => {
+        let newLastItem = data.docs[data.docs.length - 1];
+        return { newLastItem, data };
+      });
+  }
+
+  getPostColor(): string {
```

Comment:
`post.service.ts` is mainly for interacting with posts stored in Firebase. Could we move this colour into a `.scss` file and access it from there instead?

Reply:
Moving it to `utils/constants.ts` sounds good!